### ISSUE TITLE
icm20948_i2c_passthrough does not allow you to change the address with startup argument

### ISSUE THREAD

**Issue by roman-dvorak** [PX4/PX4-Autopilot#17579]
**Describe the bug**
Driver `icm20948_i2c_passthrough` does not allow you to change the address with startup argument `-a`. 

In help of driver is 
```
nsh> icm20948_i2c_passthrough
Usage: icm20948_i2c_passthrough <command> [arguments...]
 Commands:

   start
     [-I]        Internal I2C bus(es)
     [-X]        External I2C bus(es)
     [-b <val>]  board-specific bus (default=all) (external SPI: n-th bus (default=1))
     [-f <val>]  bus frequency in kHz
     [-q]        quiet startup (no message if no device found)
     [-a <val>]  I2C address
                 default: 57
     [-R <val>]  Rotation
                 default: 0
```
Where I can see that default address is 57d (0x39). It is deffined here:
https://github.com/ThunderFly-aerospace/PX4Firmware/blob/master/src/drivers/imu/invensense/icm20948/icm20948_i2c_passthrough_main.cpp#L45

```
nsh> icm20948_i2c_passthrough start -X
icm20948_i2c_passthrough #0 on I2C bus 4 (external)

nsh> icm20948_i2c_passthrough stop

nsh> icm20948_i2c_passthrough start -X -a 105
Usage: icm20948_i2c_passthrough <command> [arguments...]
 Commands:

   start
     [-I]        Internal I2C bus(es)
     [-X]        External I2C bus(es)
     [-b <val>]  board-specific bus (default=all) (external SPI: n-th bus (default=1))
     [-f <val>]  bus frequency in kHz
     [-q]        quiet startup (no message if no device found)
     [-a <val>]  I2C address
                 default: 57
     [-R <val>]  Rotation
                 default: 0

   stop

   status        print status info

nsh> i2cdetect -b 4
Scanning I2C bus: 4
     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
00: -- -- -- -- -- -- -- -- -- -- -- -- 0c -- -- -- 
10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
20: -- 21 -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
60: -- -- -- -- -- -- -- -- -- 69 -- -- -- -- -- -- 
70: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
```
If I add the  `-a 105` parameter to the command, it will show help as it did not know this argument. 

In any case, the driver uses the address hardcoded here: 
https://github.com/ThunderFly-aerospace/PX4Firmware/blob/master/src/drivers/imu/invensense/icm20948/InvenSense_ICM20948_registers.hpp#L58

```
static constexpr uint8_t I2C_ADDRESS_DEFAULT = 0b1101001;
```
> 0b1101001  0x69 105

According to the IO datasheet, this address is the right:
![image](https://user-images.githubusercontent.com/5196729/118394252-db1b5e00-b643-11eb-9bcc-33dae6cbc5f1.png)

**Expected behavior**
Expected behavior is that displayed address is one of addresses that is supported by IO and it is able to change this address with `-a` argument.

**Comment by roman-dvorak** [comment]
> Therefore driver help should list the two allowed I²C addresses e.g. 0x68 and 0x69. (Hexadecimal format is preferred because the i2cdetect also uses hex numbers).

Driver shows default address in decimal form. But you can start it with parameters in decimal or hexadecimal form (with 0x prefix).

**Comment by roman-dvorak** [comment]
solved with #17781
